src/servers/http_server.cc

```diff
@@ -203,17 +74,9 @@ class HTTPAPIServer : public HTTPServerImpl {
       const std::shared_ptr<SharedMemoryBlockManager>& smb_manager,
       const std::vector<std::string>& endpoints, const int32_t port,
       const int thread_cnt)
-      : HTTPServerImpl(port, thread_cnt), server_(server),
-        trace_manager_(trace_manager), smb_manager_(smb_manager),
-        endpoint_names_(endpoints), allocator_(nullptr),
-        api_regex_(
-            R"(/api/(health|infer|status|modelcontrol|sharedmemorycontrol|repository)(.*))"),
-        health_regex_(R"(/(live|ready))"),
-        infer_regex_(R"(/([^/]+)(?:/(\d+))?)"), status_regex_(R"(/(.*))"),
-        modelcontrol_regex_(R"(/(load|unload)/([^/]+))"),
-        sharedmemorycontrol_regex_(
-            R"(/(register|cudaregister|unregister|status)(.*))"),
-        repository_regex_(R"(/(index))")
+      : server_(server), trace_manager_(trace_manager),
+        smb_manager_(smb_manager), endpoint_names_(endpoints),
+        allocator_(nullptr), port_(port), thread_cnt_(thread_cnt)
   {
```

Comment:
Pull body of construtor and destructor outside of the class like the other functions.